src/finalizer/utils/cctp/svm/l2Tol1.ts

```diff
@@ -0,0 +1,122 @@
+import { KeyPairSigner, appendTransactionMessageInstructions, pipe, address, generateKeyPairSigner } from "@solana/kit";
+import { SVMSpokePoolClient } from "../../../../clients";
+import { CONTRACT_ADDRESSES, CCTP_MAX_SEND_AMOUNT } from "../../../../common";
+import {
+  winston,
+  SvmAddress,
+  sendAndConfirmSolanaTransaction,
+  simulateSolanaTransaction,
+  toKitAddress,
+  CHAIN_IDs,
+  getCCTPDepositAccounts,
+  createDefaultTransaction,
+  getAssociatedTokenAddress,
+  getStatePda,
+  getTransferLiabilityPda,
+  TOKEN_SYMBOLS_MAP,
+  PUBLIC_NETWORKS,
+  chainIsProd,
+  getEventAuthority,
+  createFormatFunction,
+} from "../../../../utils";
+import { SvmSpokeClient } from "@across-protocol/contracts";
+
+/**
+ * Initiates a withdrawal from the Solana spoke pool.
+ *
+ * @param solanaClient The Solana client.
+ * @param signer A base signer to be converted into a Solana signer.
+ * @param simulate Whether to simulate the transaction.
+ * @param hubChainId The chain ID of the hub.
+ * @returns A list of executed transaction signatures.
+ */
+export async function bridgeTokensToHubPool(
+  solanaClient: SVMSpokePoolClient,
+  signer: KeyPairSigner,
+  logger: winston.Logger,
+  hubChainId = 1
```

Comment:
Minor, but easy since we already import `CHAIN_IDs` here.

```suggestion
  hubChainId = CHAIN_IDs.MAINNET
```